_includes/_blogcontent/2024-03-06-introducing-scix-ambassador-program.md

```diff
@@ -0,0 +1,37 @@
+### Introducing the NASA Science Explorer Ambassador Program
+
+The [NASA Science Explorer (SciX)](https://scixplorer.org) team is thrilled to announce the launch of the SciX Ambassador Program! This initiative marks a new chapter in the evolution of the NASA Astrophysics Data System (ADS) into NASA SciX, with a significant leap in promoting interdisciplinary communication and research. To further this mission, we are seeking approximately 12 researchers to become our Lead Ambassadors, representing diverse disciplines across NASA, including Astrophysics, Planetary Science, Earth Science, Heliophysics, and Biological & Physical Sciences. Our goal is to drive forward the vision of interdisciplinary engagement and innovation as a cornerstone of NASA SciX.
+
+<div class="text-center">
+    <img class="img-thumbnail" src="{{ site.baseurl }}/blog/images/blog_2023-03-06-SciX_AmbassadorProgram_Flyer.png" />
```

Comment:
Looks like this file got renamed at some point - the src should be 
`"{{ site.baseurl }}/blog/images/SciX_AmbassadorProgram_Flyer.png`